JeMPI_Apps/JeMPI_UI/src/components/import/DropZone.tsx

```diff
@@ -189,23 +191,84 @@ const DropZone: FC = () => {
           <Typography fontSize="16px">
             <a>Click to upload</a> or drag and drop
           </Typography>
-          <Typography color="#00000099" fontSize="14px">
+          <Typography color="#00000099" fontSize="1rem">
             CSV (max. 128MB)
           </Typography>
         </div>
       </Box>
       {uploadList}
+      <Box
+        display={'flex'}
+        justifyContent="start"
+        gap={'0.5rem'}
+        alignItems="center"
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          TB 1
+        </Typography>
+        <TextField placeholder="TB 1" size="small" />
+      </Box>
+      <Box
+        display={'flex'}
+        justifyContent="start"
+        gap={'0.5rem'}
+        alignItems={'center '}
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          CB
+        </Typography>
+        <FormControlLabel
+          value="CB2"
+          control={<Checkbox />}
+          label="CB1"
+          labelPlacement="start"
+        />
+        <FormControlLabel
+          value="CB2"
+          control={<Checkbox />}
+          label="CB2"
+          labelPlacement="start"
+        />
+      </Box>
+      <Box
+        display={'flex'}
+        gap={'1.3rem'}
+        alignItems={'center'}
+        padding={'0.5rem'}
+      >
+        <Typography fontWeight={'bold'} fontSize={'1.2rem'}>
+          RB
+        </Typography>
+        <FormControl>
+          <RadioGroup
+            aria-labelledby="demo-radio-buttons-group-label"
+            defaultValue="Opt1"
+            name="radio-buttons-group"
+          >
+            <FormControlLabel value="Opt1" control={<Radio />} label="Opt1" />
+            <FormControlLabel value="Opt2" control={<Radio />} label="Opt2" />
+            <FormControlLabel value="Opt3" control={<Radio />} label="Opt3" />
+          </RadioGroup>
+        </FormControl>
+      </Box>
       <CardActions
         sx={{ display: 'block', textAlign: 'center', marginTop: '5%' }}
       >
         <Button
           variant="contained"
           onClick={handleUpload}
-          disabled={uploadFileMutation.isLoading}
+          disabled={
```

Comment:
Should we not tell the user why the button is disabled. i.e `Previous file is still being uploaded` least they be confused?

Reply:
Oh, okay... so if you refresh the page whilst an upload is happening, the progress bar will be redrawn showing the state?

Reply:
the code that refers to the state is actually junk code, I cleaned the code in this commit 
https://github.com/jembi/JeMPI/pull/126/commits/54a1cc053c59b4cbcdceed371f5a3b7cd8944729
the state actually has pending and in progress, once it fails or completes the interface back to the upload zone UI with a toast carries information for users.